From: Jan Engelhardt <jengelh@inai.de>
Date: 2017-11-28 11:01:12.978019538 +0100

Kill /usr/bin/env. We can't have random python interpreters be executed.
---
 tools/libinput-analyze-per-slot-delta.py    |    2 +-
 tools/libinput-analyze-recording.py         |    2 +-
 tools/libinput-analyze-touch-down-state.py  |    2 +-
 tools/libinput-measure-fuzz.py              |    2 +-
 tools/libinput-measure-touch-size.py        |    2 +-
 tools/libinput-measure-touchpad-pressure.py |    2 +-
 tools/libinput-measure-touchpad-size.py     |    2 +-
 tools/libinput-measure-touchpad-tap.py      |    2 +-
 tools/libinput-record-verify-yaml.py        |    2 +-
 tools/libinput-replay.py                    |    2 +-
 tools/test_tool_option_parsing.py           |    2 +-
 11 files changed, 11 insertions(+), 11 deletions(-)

Index: libinput-1.17.0/tools/libinput-analyze-per-slot-delta.py
===================================================================
--- libinput-1.17.0.orig/tools/libinput-analyze-per-slot-delta.py
+++ libinput-1.17.0/tools/libinput-analyze-per-slot-delta.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8
 # vim: set expandtab shiftwidth=4:
 # -*- Mode: python; coding: utf-8; indent-tabs-mode: nil -*- */
Index: libinput-1.17.0/tools/libinput-analyze-recording.py
===================================================================
--- libinput-1.17.0.orig/tools/libinput-analyze-recording.py
+++ libinput-1.17.0/tools/libinput-analyze-recording.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8
 # vim: set expandtab shiftwidth=4:
 # -*- Mode: python; coding: utf-8; indent-tabs-mode: nil -*- */
Index: libinput-1.17.0/tools/libinput-analyze-touch-down-state.py
===================================================================
--- libinput-1.17.0.orig/tools/libinput-analyze-touch-down-state.py
+++ libinput-1.17.0/tools/libinput-analyze-touch-down-state.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8
 # vim: set expandtab shiftwidth=4:
 # -*- Mode: python; coding: utf-8; indent-tabs-mode: nil -*- */
Index: libinput-1.17.0/tools/libinput-measure-fuzz.py
===================================================================
--- libinput-1.17.0.orig/tools/libinput-measure-fuzz.py
+++ libinput-1.17.0/tools/libinput-measure-fuzz.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # vim: set expandtab shiftwidth=4:
 # -*- Mode: python; coding: utf-8; indent-tabs-mode: nil -*- */
 #
Index: libinput-1.17.0/tools/libinput-measure-touch-size.py
===================================================================
--- libinput-1.17.0.orig/tools/libinput-measure-touch-size.py
+++ libinput-1.17.0/tools/libinput-measure-touch-size.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # vim: set expandtab shiftwidth=4:
 # -*- Mode: python; coding: utf-8; indent-tabs-mode: nil -*- */
 #
Index: libinput-1.17.0/tools/libinput-measure-touchpad-pressure.py
===================================================================
--- libinput-1.17.0.orig/tools/libinput-measure-touchpad-pressure.py
+++ libinput-1.17.0/tools/libinput-measure-touchpad-pressure.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # vim: set expandtab shiftwidth=4:
 # -*- Mode: python; coding: utf-8; indent-tabs-mode: nil -*- */
 #
Index: libinput-1.17.0/tools/libinput-measure-touchpad-size.py
===================================================================
--- libinput-1.17.0.orig/tools/libinput-measure-touchpad-size.py
+++ libinput-1.17.0/tools/libinput-measure-touchpad-size.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # vim: set expandtab shiftwidth=4:
 # -*- Mode: python; coding: utf-8; indent-tabs-mode: nil -*- */
 #
Index: libinput-1.17.0/tools/libinput-measure-touchpad-tap.py
===================================================================
--- libinput-1.17.0.orig/tools/libinput-measure-touchpad-tap.py
+++ libinput-1.17.0/tools/libinput-measure-touchpad-tap.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # vim: set expandtab shiftwidth=4:
 # -*- Mode: python; coding: utf-8; indent-tabs-mode: nil -*- */
 #
Index: libinput-1.17.0/tools/libinput-record-verify-yaml.py
===================================================================
--- libinput-1.17.0.orig/tools/libinput-record-verify-yaml.py
+++ libinput-1.17.0/tools/libinput-record-verify-yaml.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # vim: set expandtab shiftwidth=4:
 # -*- Mode: python; coding: utf-8; indent-tabs-mode: nil -*- */
 #
Index: libinput-1.17.0/tools/libinput-replay.py
===================================================================
--- libinput-1.17.0.orig/tools/libinput-replay.py
+++ libinput-1.17.0/tools/libinput-replay.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # vim: set expandtab shiftwidth=4:
 # -*- Mode: python; coding: utf-8; indent-tabs-mode: nil -*- */
 #
Index: libinput-1.17.0/tools/test_tool_option_parsing.py
===================================================================
--- libinput-1.17.0.orig/tools/test_tool_option_parsing.py
+++ libinput-1.17.0/tools/test_tool_option_parsing.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # vim: set expandtab shiftwidth=4:
 # -*- Mode: python; coding: utf-8; indent-tabs-mode: nil -*- */
 #
